src/PrivateRoute.tsx

```diff
@@ -12,7 +12,7 @@ const useIsWalletConnected = (): {
   useEffect(() => {
     import(
       /* webpackChunkName: "API_chunk"*/
-      'api'
+      'apps/gp-v1/api'
     )
       .then(({ walletApi }) => walletApi.isConnected())
       .then((isConnected) => {
```

Comment:
Next refactor step would be to split the hooks between shared/common and app specific.

Then we can import common from base /api instances and so on.